lua/autorun/owoifier.lua

```diff
@@ -0,0 +1,72 @@
+local OwOemotes = {"OwO", "owo", "UwU", "uwu", "TwT", ">w<", ">wO", "Ow<", ":3c", ":3", ":33", ":333", "nya", "(> owo )>",
```

Comment:
all this code is serverside, so it should just be in autorun/server without any SERVER checks